cmd/kube-apiserver/app/server.go

```diff
@@ -210,9 +212,16 @@ func Run(s *options.APIServer) error {
 				installSSH = instances.AddSSHKeyToAllInstances
 			}
 		}
-
+		if ports.KubeletReadOnlyPort <= 0 {
+			glog.Fatalf("Must enable kubelet read-only port if proxy ssh-tunneling is specified.")
+		}
 		// Set up the tunneler
-		tunneler = master.NewSSHTunneler(s.SSHUser, s.SSHKeyfile, installSSH)
+		healthCheckPath := &url.URL{
+			Scheme: "https",
```

Comment:
Insecure (to not care about the signer of kubelet's cert) is set on the Transport. Is that what you were asking about?